VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/tests/waiting-room-base-setup.ts

```diff
@@ -29,6 +29,10 @@ export const participantsWitness = conferenceTestData.getListOfParticipantsWitne
 participantsWitness.forEach(x => {
     globalConference.participants.push(x);
 });
+export const participantsPanelMemebers = conferenceTestData.getListOfParticipantsWingers();
```

Comment:
where's this used?

Reply:
for the test to determine num of judicial in a consultation